app/main.py

```diff
@@ -2,0 +1,34 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
```

Comment:
Code Style: Use interpolation instead of concatenation. Instead of using "Name:" + self.name, you should use f-string like this: f"Name: {self.name}". This makes your code cleaner and easier to read.